saleor/graphql/account/mutations/base.py

```diff
@@ -26,6 +25,8 @@
 )
 from ...core.types.common import AccountError
 
```

Comment:
Remove new line.

Reply:
This was not altered. Should the line be removed anyway?